core/src/main/java/hudson/util/LogTaskListener.java

```diff
@@ -82,12 +70,13 @@ public void close() {
         private final StackTraceElement caller;
         private final ByteArrayOutputStream baos = new ByteArrayOutputStream();
 
-        public LogOutputStream(Logger logger, Level level, StackTraceElement caller) {
+        LogOutputStream(Logger logger, Level level, StackTraceElement caller) {
```

Comment:
🐛  - Potential breaker for code depending on this constructor.

Reply:
Not true; the class was `private` already, so the `public` modifier on its constructor was just misleading.